docs/3.0-migration-guide.d/combine_SSL_CID-TLS1_3_PADDING_GRANULARITY_options.md

```diff
@@ -0,0 +1,14 @@
+Combine the `MBEDTLS_SSL_CID_PADDING_GRANULARITY` and `MBEDTLS_SSL_TLS1_3_PADDING_GRANULARITY` options
+--
+
+This change affects users who modified the default `config.h` padding granularity
+settings, i.e. enabled at least one of the options.
+
+`config.h` options `MBEDTLS_SSL_CID_PADDING_GRANULARITY` and
```

Comment:
Minor: add "The" at the beginning to make the sentence start with a normal word.

Reply:
Done